gtsam/basis/basis.i

```diff
@@ -137,7 +137,7 @@ class FitBasis {
   static gtsam::GaussianFactorGraph::shared_ptr LinearGraph(
       const std::map<double, double>& sequence,
       const gtsam::noiseModel::Base* model, size_t N);
-  This::Parameters parameters() const;
+  gtsam::This::Parameters parameters() const;
```

Comment:
This (no pun intended) doesn't make sense to me - is `This` a special keyword in the wrapper or something?

Shouldn't this make sense to just be `gtsam::FitBasis::Parameters` ?

Reply:
This is used when we have templated types so that the correct substitution for the template happens.